package.json

```diff
@@ -2,20 +2,6 @@
   "name": "call-home-mvp",
   "version": "1.0.0",
   "description": "This is very much in the MVP stage.",
-  "scripts": {
-    "build": "docker build . -t call-home",
-    "deploy:cloudrun": "./buildAndDeploy.sh cloud-run",
-    "deploy:staging": "./buildAndDeploy.sh staging",
-    "deploy:prod": "./buildAndDeploy.sh prod",
-    "init": "npm run init:shared && npm run init:backend && npm run init:frontend",
-    "init:shared": "cd shared && npm run init",
-    "init:frontend": "cd frontend && npm run init",
-    "init:backend": "cd backend && npm run init",
-    "precommit:shared": "cd shared && npm run precommit",
-    "precommit:frontend": "cd frontend && npm run precommit",
-    "precommit:backend": "cd backend && npm run precommit",
-    "precommit": "npm run precommit:frontend && npm run precommit:backend && npm run precommit:shared"
-  },
   "repository": {
```

Comment:
All of the scripts have been removed and the new ones have no documentation. Could you add documentation so that devs whose workflows have to change (e.g. me 😛 ) know what they should be using?

Reply:
Modified the README.md and added some documentation to the Makefile,